packages/block-library/src/term-count/block.json

```
@@ -0,0 +1,64 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"__experimental": true,
+	"name": "core/term-count",
+	"title": "Term Count",
+	"category": "theme",
+	"description": "Displays the count of a taxonomy term.",
+	"textdomain": "default",
+	"usesContext": [ "termId", "taxonomy" ],
+	"attributes": {
+		"textAlign": {
+			"type": "string"
+		},
+		"hasParenthesis": {
+			"type": "boolean",
+			"default": true
+		}
+	},
+	"supports": {
+		"align": [ "wide", "full" ],
+		"html": false,
+		"color": {
+			"gradients": true,
+			"link": true,
+			"__experimentalDefaultControls": {
+				"background": true,
+				"text": true,
+				"link": true
```

```suggestion
				"text": true,
```

This block doesn't render any `a` element.